### DISCUSSION TITLE
Reset Questions

### DISCUSSION THREAD

**Discussion by kykym** [DavidS95/Smokeless_UMAF#12] [Q&A]
Hello and thank you for your tool
If it's possible I'd like you'll clarify some things

1) If I use a reset option, what settings would load up? One that were put by the manufacturer into my BIOS or some other settings that were in the files that I downloaded?
2) If I remove the battery, will it reset everything completely to the manufacturer's defaults? Will taking out battery or resetting CMOS by the other way help if I brick my device by doing something wrong with this tool?
3) What does tthe sentence 
"Curve Optimizer for certain apus commonly unlocked 5000 series - however this can be done through windows using UXTU for all affected models" mean?
I have R5 5600H, so it's dangerous for me to use the tool's gui bios curve optimizer? I should use the  one that run in Windows instead? Or what?

**Comment by Cynthianon** [comment]
I can only answer your question number 2. And from my experience just now you can't reset a bricked device by unplugging the battery and drain the remaining power. My laptop doesn't have a CMOS battery so I can't tell if that would help however.

**Reply by kykym** [reply]
Thanks. You say that your laptop doesn't have cmos, but haven't you written that you located the CMOS battery in your thread? If you have CMOS, main battery can't interfere with BIOS in any way...

**Reply by Cynthianon** [reply]
I'm not that knowledgeable about it, the CMOS I located was the memory chip. I searched for its serial number on Google and I found it's a non volatile memory, meaning it doesn't have a CMOS battery with it. So that's why I was able to locate the CMOS chip but not the CMOS battery, and also why disconnecting the main battery did nothing.
Maybe CMOS does specifically refer to a volatile memory type, in that case I'm sorry for the confusion.

**Comment by DavidS95** [comment]
1. I assume bios default options would be set.
2. If removing the battery does a cmos clear that will reset the settings, if your laptop has a cmos battery you will need to remove that, however if it doesnt the internal battery is what you pullout for a reset.
3.  5600h cannot use curve optimizer at all. That sentence if for all unlocked 5000 series apus, since CO only works and shows up on them. If you have CO and you are a 5000 apu, you need to use it through windows as setting it in bios does on some models prevent boot.

**Reply by kykym** [reply]
Thank you for a complex answer
By the way I was under the Impression that UXTU Curve doesn't work on 5000's APU, as it was stated in their chat multiple times.

**Reply by DavidS95** [reply]
Thats because that is correct. On 99% of 5000 series apus they cannot use CO. The only ones that can are the HX and hs processors which make up the ryzen 9s. So your 5600h cant but if it was a 5980hx u could despite both being 5000 series.

**Comment by kykym** [comment]
update: writing new bios image through HP tools resets changes in unlocked BIOS visually, but not for real, I had my core performance boost changed to auto visually, but the frequency is still locked
